votor/src/consensus_pool.rs

```diff
@@ -1030,14 +1021,14 @@ mod tests {
         assert!(pool.make_start_leader_decision(my_leader_slot, parent_slot, first_alpenglow_slot));
     }
 
-    #[test_case(Vote::new_finalization_vote(5), vec![CertificateType::Finalize])]
-    #[test_case(Vote::new_notarization_vote(6, Hash::new_unique()), vec![CertificateType::Notarize, CertificateType::NotarizeFallback])]
-    #[test_case(Vote::new_notarization_fallback_vote(7, Hash::new_unique()), vec![CertificateType::NotarizeFallback])]
-    #[test_case(Vote::new_skip_vote(8), vec![CertificateType::Skip])]
-    #[test_case(Vote::new_skip_fallback_vote(9), vec![CertificateType::Skip])]
+    #[test_case(Vote::new_finalization_vote(5), vec![CertificateType::Finalize(5)])]
+    #[test_case(Vote::new_notarization_vote(6, Hash::default()), vec![CertificateType::Notarize(6, Hash::default()), CertificateType::NotarizeFallback(6, Hash::default())])]
```

Comment:
Why change all Hash::new_unique() to Hash::default()?

We probably don't test that we are passing block_id correctly, but we should. Making it Hash::default() everywhere makes it impossible to verify.

Reply:
This is now addressed in https://github.com/anza-xyz/agave/pull/8701.